packages/langchain/src/openai/types.ts

```diff
@@ -0,0 +1,64 @@
+import type { BaseChatModelParams } from '@langchain/core/language_models/chat_models';
+import { BaseLLMParams } from '@langchain/core/language_models/llms';
+import type {
+  ChatOpenAICallOptions,
+  OpenAIChatInput,
+  OpenAIEmbeddingsParams
+} from '@langchain/openai';
+import type {
+  OpenAiChatCompletionParameters,
+  OpenAiChatModel,
+  OpenAiEmbeddingModel
+} from '@sap-ai-sdk/foundation-models';
+import type { ConfigurationOptions } from '@sap-ai-sdk/ai-api';
+
+/**
+ * Input for Text generation for OpenAI GPT.
+ */
+export type OpenAIChatModelInput = Omit<
```

Comment:
[nth] A comment why we omit certain properties here would help. If I remember correctly it was because in langchain they are required but not in our azure openai client (or maybe the other way around), but it's not obvious right away.